tools/cockpit.spec

```diff
@@ -551,6 +559,11 @@ if %{_sbindir}/selinuxenabled 2>/dev/null; then
 fi
 %endif
 %systemd_postun_with_restart cockpit.socket cockpit.service
+# clean up dynamic motd/issue symlinks on removal
```

Comment:
Is this really needed? I read in http://ftp.rpm.org/max-rpm/s1-rpm-inside-files-list-directives.html that ghost files are also removed when the package is.

Reply:
Excellent point -- I'll test this! (easy enough with the integration tests now). If it does, I'll drop that bit. Setting needswork for now.